packages/web-react/src/components/Pagination/README.md

```diff
@@ -0,0 +1,153 @@
+# Pagination
+
+## Example with the current item in first place
+
+```jsx
+<Pagination>
+  <PaginationItem>
+    <PaginationLink href="#" isCurrent accessibilityLabel="Current Page, Page 1" pageNumber={1} />
+  </PaginationItem>
+  <PaginationItem>
+    <PaginationLink href="#" accessibilityLabel="Goto Page 2" pageNumber={2} />
+  </PaginationItem>
+  ...
+  <PaginationItem>
+    <PaginationLinkNext href="#" elementType="a" />
+  </PaginationItem>
+</Pagination>
+```
+
+## Example with the current ite in the middle
+
+```jsx
+<Pagination>
+  <PaginationItem>
+    <PaginationLinkPrevious href="#" elementType="a" />
+  </PaginationItem>
+  <PaginationItem>
+    <PaginationLink href="#" accessibilityLabel="Goto Page 11" pageNumber={11} />
+  </PaginationItem>
+  ...
```

Comment:
Especially in case of a component like this, I'd turn all these dots into comments:

```suggestion
  <!-- ... -->
```

Otherwise, developers could think the ellipsis is part of the composition which wouldn't be unusual for pagination.

Reply:
As far as I know, the README is not rendered anywhere, so the comment here is pointless, this is how we use it for more component examples and then there is one more thing and that is the comment format. It's fine for HTML, but in JSX this will be rendered exactly for you. As for comments in React, that is `{/* jsx comment */}` `<!-- not jsx comment -->`

Reply:
Fixed with `{'...'}`